src/results/results.jl

```diff
@@ -111,6 +111,18 @@ function reopt_results(m::JuMP.AbstractModel, p::REoptInputs; _n="")
     if "ASHPWaterHeater" in p.techs.ashp_wh
         add_ashp_wh_results(m, p, d; _n)
     end
+
+    d["Financial"]["year_one_fuel_cost_before_tax"] = 0.0
+    d["Financial"]["year_one_fuel_cost_after_tax"] = 0.0
+    for tech in p.techs.fuel_burning
+        if tech in keys(d)
+            d["Financial"]["year_one_fuel_cost_before_tax"] += d[tech]["year_one_fuel_cost_before_tax"]
+            d["Financial"]["year_one_fuel_cost_after_tax"] += d[tech]["year_one_fuel_cost_after_tax"]
+        end
+    end
+    
+    d["Financial"]["year_one_total_cost_before_tax"] = d["ElectricTariff"]["year_one_bill_before_tax"] - d["ElectricTariff"]["year_one_export_benefit_before_tax"] + d["Financial"]["year_one_chp_standby_cost_before_tax"] + d["Financial"]["year_one_fuel_cost_before_tax"] + d["Financial"]["year_one_om_costs_before_tax"]
```

Comment:
Especially since this is being added to the financial results outputs, it could be helpful to change the name to indicate it does not include CAPEX

Reply:
How about year_one_total_operating_cost_after_tax?